guidelines/index.html

```diff
@@ -1403,30 +1403,30 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
+             <section class="guideline" id="focus-not-obscured-minimum"><div class="header-wrapper"><h4 id="x2-4-11-focus-not-obscured-minimum"><bdi class="secno">達成基準 2.4.11 </bdi>隠されないフォーカス (最低限)</h4><a class="self-link" href="#focus-not-obscured-minimum" aria-label="Permalink for Section 2.4.11"></a></div>
    
     
     
     <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html">Understanding Focus Not Obscured (Minimum)</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#focus-not-obscured-minimum">How to Meet Focus Not Obscured (Minimum)</a></div><p class="conformance-level">(レベル AA)</p>
 	<p class="change">[New]</p>
     
-    <p>When a <a href="#dfn-user-interface-components" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-interface-components-7" title="コンテンツの一部分で、特定の機能を実現するための単一のコントロールとして利用者が知覚するもの。">user interface component</a> receives keyboard focus, the component is not entirely hidden due to author-created content.</p>
+    <p><a href="#dfn-user-interface-components" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-interface-components-7" title="コンテンツの一部分で、特定の機能を実現するための単一のコントロールとして利用者が知覚するもの。">ユーザインタフェース コンポーネント</a>がキーボードフォーカスを受け取るとき、コンテンツ制作者が作成したコンテンツによって、そのコンポーネントが完全には隠されない。</p>
 
-    <div class="note" role="note" id="issue-container-generatedID-27"><div role="heading" class="note-title marker" id="h-note-27" aria-level="5"><span>注記 1</span></div><p class="">Where content in a configurable interface can be repositioned by the user, then only the initial positions of user-movable content are considered for testing and conformance of this Success Criterion.</p></div>
+    <div class="note" role="note" id="issue-container-generatedID-27"><div role="heading" class="note-title marker" id="h-note-27" aria-level="5"><span>注記 1</span></div><p class="">設定可能なインタフェースの中にあるコンテンツが利用者によって再配置可能な場合、利用者によって移動できるコンテンツの初期位置のみが、この達成基準のテスト及び適合性の対象として考慮に入れられる。</p></div>
 
-    <div class="note" role="note" id="issue-container-generatedID-28"><div role="heading" class="note-title marker" id="h-note-28" aria-level="5"><span>注記 2</span></div><p class="">Content opened by the <em>user</em> may obscure the component receiving focus. If the user can reveal the focused component without advancing the keyboard focus, the component with focus is not considered hidden due to author-created content.</p></div>
+    <div class="note" role="note" id="issue-container-generatedID-28"><div role="heading" class="note-title marker" id="h-note-28" aria-level="5"><span>注記 2</span></div><p class=""><em>利用者</em>によって開かれたコンテンツが、フォーカスを受け取るコンポーネントを隠してしまうことがあるかもしれない。利用者がキーボードフォーカスを進めることなく、フォーカスされたコンポーネントを見えるようにできる場合は、そのフォーカスされたコンポーネントは、コンテンツ制作者が作成したコンテンツによって隠されたとはみなされない。</p></div>
```

Comment:
態をいじくり回してみましたがどうでしょうか…。
```suggestion
    <div class="note" role="note" id="issue-container-generatedID-28"><div role="heading" class="note-title marker" id="h-note-28" aria-level="5"><span>注記 2</span></div><p class=""><em>利用者</em>が開いたコンテンツによって、フォーカスを受け取るコンポーネントが隠される場合がある。利用者がキーボードフォーカスを移動せずにフォーカスを持つコンポーネントを表示できる場合、そのフォーカスを持つコンポーネントは、コンテンツ制作者が作成したコンテンツによって隠されるとはみなされない。</p></div>
```